src/hotspot/cpu/riscv/stubGenerator_riscv.cpp

```diff
@@ -4032,9 +4030,8 @@ class StubGenerator: public StubCodeGenerator {
               int k, int s, int t,
               Register rtmp1, Register rtmp2, Register rmask32) {
     // rtmp1 = c ^ (b | (~d))
-    __ notr(rtmp2, d);
-    __ orr(rtmp1, b, rtmp2);
-    __ xorr(rtmp1, c, rtmp1);
+    __ ornr(rtmp2, b, d);
```

Comment:
Nit: You could use `rtmp1` here, the dependency for the next instruction will block the CPU from executing OOO anyway.

Reply:
Done.